clang-tools-extra/docs/clang-tidy/checks/readability/use-span-first-last.rst

```diff
@@ -0,0 +1,24 @@
+.. title:: clang-tidy - readability-use-span-first-last
+
+readability-use-span-first-last
+===============================
+
+Suggests using ``std::span::first()`` and ``std::span::last()`` member functions 
+instead of equivalent ``subspan()``. These dedicated methods were added to C++20 
+to provide more expressive alternatives to common subspan operations.
+
+Covered scenarios:
+
+=============================== ============
```

Comment:
Please reduce first column on 4 characters and add two characters to second column, so all cell content will be covered.

Reply:
resolved.